addons/multus/multus.yaml

```diff
@@ -123,17 +136,33 @@ spec:
             limits:
               cpu: "100m"
               memory: "50Mi"
-          securityContext:
-            allowPrivilegeEscalation: true
           volumeMounts:
             - name: cni
               mountPath: /host/etc/cni/net.d
             - name: cnibin
               mountPath: /host/opt/cni/bin
+      initContainers:
+        - name: install-multus-binary
+          image: ghcr.io/k8snetworkplumbingwg/multus-cni:v3.8.1
+          command:
+            - "cp"
+            - "/usr/src/multus-cni/bin/multus"
+            - "/host/opt/cni/bin/multus"
+          resources:
+            requests:
+              cpu: "10m"
+              memory: "15Mi"
+          securityContext:
+            privileged: true
+          volumeMounts:
+            - name: cnibin
+              mountPath: /host/opt/cni/bin
+              mountPropagation: Bidirectional
```

Comment:
Why is `mountPropagation: Bidirectional`  needed?

Reply:
Hello
This setting actually comes from the configuration published by the [upstream project](https://github.com/k8snetworkplumbingwg/multus-cni/blob/8bbb594dad15d8c617fccc21b9409de21686ef40/deployments/multus-daemonset.yml#L211=).
My understanding is that Multus handles the configuration of the network interfaces, as part of that it has to execute various binaries (CNI plugin) depending on the configuration requested, so if a new CNI plugin is installed on Microk8s it has to be immediately available to Multus. At the same time, Multus installs its own binary on this directory [at startup](https://github.com/k8snetworkplumbingwg/multus-cni/blob/8bbb594dad15d8c617fccc21b9409de21686ef40/images/entrypoint.sh#L203-L205=) (I'm not sure of the reason though, maybe this is for consistency) so this mount point needs to be available from both side.

Reply:
Explanation makes sense, but I would expect the `mountPropagation: Bidirectional` to be used in the volume mount for the container, not the initContainer instead.

Anyway, I did not find anything relevant in https://github.com/k8snetworkplumbingwg/multus-cni/pull/718, so I'm fine with this. Thanks!